src/promptflow-devkit/promptflow/_sdk/_tracing.py

```diff
@@ -441,7 +400,10 @@ def start_trace_with_devkit(collection: str, **kwargs: typing.Any) -> None:
     # print tracing url(s) when run is specified
     _print_tracing_url_from_local(pfs_port=pfs_port, collection=collection, exp=exp, run=run)
     if ws_triad is not None and is_azure_ext_installed:
-        _print_tracing_url_from_azure_portal(ws_triad=ws_triad, collection=collection, exp=exp, run=run)
+        print(
```

Comment:
maybe we don't need this print anymore?

Reply:
attach one screenshot in pr description?

Reply:
It's a reminder to notify user that the cloud trace is indeed enabled

Reply:
make sense

Reply:
updated pr description